test/parallel/test-repl-.editor.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const repl = require('repl');
+
+const terminalCode = '\u001b[1G\u001b[0J> \u001b[3G';
```

Comment:
Perhaps a comment here, explaining what these characters are, would be of much help.